.drone.yml

```diff
@@ -4,9 +4,17 @@ build:
         - /home/ubuntu/android_build/android-sdk:/usr/local/android-sdk
         - /home/ubuntu/.android:/root/.android
     commands:
-        - ./gradlew build
+        - ./gradlew build javadocJar sourcesJar
+        - ./gradlew rxlifecycle:aarWrapper
+        - ./gradlew todo:aarWrapper
 publish:
-    script:
-        commands:
-            - ./gradlew todo:artPub
-            - ./gradlew rxlifecycle:artPub
+  artifactory:
+    url: $$ARTIFACTORY_URL
+    username: $$ARTIFACTORY_USER
+    password: $$ARTIFACTORY_PASSWORD
+    pom: todo/build/publications/aarWrapper/pom-default.xml 
```

Comment:
so all this for just the 'Todo' check.  I still say we don't auto-publish these.  It's not hard once it's ready to run `artPub` for the one sub-project once it's ready.  Besides, you still have to go into artifactory and push it to bintray manually, yes?